docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - "8180:8180"
     volumes:
       - './configs/peer:/config'
-    init: true
+  #  init: true
```

Comment:
What is the reason to comment out? This reverts the result of #2311

Reply:
I apologize, it's was caused by playing #2594 (I forgot to change the branch).